cohorts/survival.py

```diff
@@ -76,6 +80,38 @@ def plot_kmf(df,
         print_as_title: bool, optional, whether or not to print text
           within the plot's title vs. stdout, default False
     """
+    # if strata not None, call plot_kmf for each of the strata
+    if strata_col:
+        # want this at the top of the function!
+        arglist = locals() ## save input args for reuse
```

Comment:
Yeah, I was trying to avoid copying & pasting all the parameters that many times .. feels a bit repetitive but may be preferable ... Your point about modifying the locals() dict is really spot on, .. I forget that this copies by reference not value. It general it seems this isn't a common pattern in Python, so will avoid it!